syntax = "proto3";

// The greeting service definition.
service Greeter {
  // Sends a greeting
  rpc SayHello (HelloRequest) returns (HelloReply) {}
}

// The request message containing the user's name,
// as well as a JSON string storing the span context
message HelloRequest {
  string spanContext = 1;
  string name = 2;
}

// The response message containing the greetings
message HelloReply {
  string message = 1;
}
